tracing/jaeger/src/main/java/io/helidon/tracing/jaeger/JaegerTracerBuilder.java

```diff
@@ -480,6 +511,22 @@ boolean isEnabled() {
         return enabled;
     }
 
+    private static ContextPropagators createPropagators(Set<PropagationFormat> propagatorFormats) {
+        List<TextMapPropagator> propagators = new ArrayList<>();
+
+        for (PropagationFormat propagatorFormat : propagatorFormats) {
+            switch (propagatorFormat) {
+
+            case B3 -> propagators.add(B3Propagator.injectingMultiHeaders());
+            case W3C -> propagators.add(W3CBaggagePropagator.getInstance());
+            // jaeger and unknown are jaeger
+            default -> propagators.add(JaegerPropagator.getInstance());
+            }
+        }
```

Comment:
```suggestion
List<TextMapPropagator> propagators = Arrays.stream(propagatorFormats)
    .map(propagatorFormat -> {
        switch (propagatorFormat) {
            case B3:
                return B3Propagator.injectingMultiHeaders();
            case W3C:
                return W3CBaggagePropagator.getInstance();
            // jaeger and unknown are jaeger
            default:
                return JaegerPropagator.getInstance();
        }
    })
    .collect(Collectors.toList());
```

Reply:
Just a suggestion :)

Reply:
I have done it slightly differently, though this was a very good comment, thanks